src/evrythng/tests/test_entities.py

```diff
@@ -0,0 +1,82 @@
+import unittest
+import os
+from evrythng.entities import actions
+from evrythng.entities import products
+
+API_KEY = os.environ['API_KEY']
+
+
+class TestProductMethods(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        """Product Tests Setup"""
+
+        cls.product_id = products.create_product(
+            name='test_creating_prod_GooeeIOT', api_key=API_KEY).json()['id']
+
+    @classmethod
+    def tearDownClass(cls):
+        """Product Tests Destroying"""
+
+        products.delete_product(
+            product_id=str(cls.product_id), api_key=API_KEY)
+
+    def test1_creation(self):
+        """creating test product and
+         check if existing name is equal to test str"""
```

Comment:
Check out the official PEP for docstrings: https://www.python.org/dev/peps/pep-0257/
That's our standard for docstrings :)